noxfile.py

```diff
@@ -288,7 +288,7 @@ def prerelease_deps(session):
         "google-cloud-bigquery-storage",
         "google-cloud-core",
         "google-resumable-media",
-        "grpcio",
+        "grpcio!=1.49.0rc1",
```

Comment:
If it can't be higher than 1.49.0rc1, wouldn't it be better to note it with `grpcio<=1.49.0rc1`?

Reply:
It is allowed to be higher than `1.49` (presuming future releases fix the problem.)
For now, across a number of googleapis python packages, it appears that this selective approach of avoiding just this release candidate is the approach that is being used. In fact, it appears that this approach has already been applied to this repo (and this one line change will be rendered moot).
https://github.com/googleapis/python-bigquery/pull/1328/files

Reply:
I think it should be good to revert this then.